### PR TITLE
chore(main): release 1.1.0

### PR DESCRIPTION
**Pull request by johmsalas**
:robot: I have created a release *beep* *boop*
---

### Features

* **Subs:** adjustable name for substitude command ([25f965c](https://github.com/johmsalas/text-case.nvim/commit/25f965c9a6f440b1a9f4cfb6388257f227945652))
* **Subs:** adjustable name for substitude command ([25f965c](https://github.com/johmsalas/text-case.nvim/commit/25f965c9a6f440b1a9f4cfb6388257f227945652))
* **Subs:** adjustable name for substitude command ([4cc7275](https://github.com/johmsalas/text-case.nvim/commit/4cc727581cfb090dd1842caa478d23afc0af516f))

---
This PR was generated with [Release Please](https://github.com/googleapis/release-please). See [documentation](https://github.com/googleapis/release-please#release-please).

### CODE REVIEW

**Comment by peterfication** [comment]
I think I have to use squash merge from now on so Release Please is happy.

**Comment by johmsalas** [comment]
@peterfication no worries, we can commit some adjustments before merging it
TIL: it ignores "docs" commits, https://github.com/johmsalas/text-case.nvim/pull/130. Which makes sense for release notes. I thought it was broken until there was this PR including your changes

**Comment by peterfication** [comment]
Should I adjust the changelog then so that we can merge it and release 1.1.0?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## [1.1.0](https://github.com/johmsalas/text-case.nvim/compare/v1.0.0...v1.1.0) (2023-12-20)
```
Comment (johmsalas) [inline CHANGELOG.md:3]: @peterfication should we release for every new change? :thinking:

Comment (johmsalas) [reply CHANGELOG.md:3]: I guess it will be also CI/CD now :thought_balloon:

Comment (peterfication) [reply CHANGELOG.md:3]: Yeah, I was asking myself this as well ...

Comment (johmsalas) [reply CHANGELOG.md:3]: Let's do it for this one! I will start observing how other plugins do
You can approve and merge

**Review by peterfication — APPROVED** [review]

**Comment by johmsalas** [comment]
:robot: Release is at https://github.com/johmsalas/text-case.nvim/releases/tag/v1.1.0 :sunflower:

**Pull request merged**